python/dgl/distributed/kvstore.py

```diff
@@ -1119,6 +1119,25 @@ def get_data_meta(self, name):
         part_policy = self._part_policy[name]
         return (data_type, data_shape, part_policy)
 
+    def get_partid(self, name, id_tensor):
```

Comment:
do we need this API? we can get the same information from the graph partition book.

Reply:
Which API should I use?

Reply:
i see. you want to have a global place for mapping IDs to part IDs. GraphPartitionBook requires to know if it's mapping node IDs or edge IDs.